server/src/test/java/org/schoellerfamily/pathfinderhelper/test/PathfinderHelperApplicationTests.java

```diff
@@ -0,0 +1,16 @@
+package org.schoellerfamily.pathfinderhelper.test;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class PathfinderHelperApplicationTests {
+
```

Comment:
As you can see, right now there are no tests in the test class. That's not marvelous.